### DISCUSSION TITLE
#49 togETHer biweekly meeting

### DISCUSSION THREAD

**Discussion by Emilyxiannv** [tog-ETH-er/togETHer#31] [General]
Agenda:

Community updates
Fireside discussion

会议主题：togETHer
会议时间： 21:00-22:00
重复周期：每两周 (周一)

点击链接入会，或添加至会议列表：
https://meeting.tencent.com/s/LxNajKWJw4RV

会议 ID：891 5098 6775

**Comment by bambalados** [comment]
**Crypto Tech Night**

- SBT Twitter Space 和 vitae3 & TampabayDAO - 10.24号
- 规划做市商与安全的活动

**Comment by luckyyang** [comment]
### Rebase
- web3 daily 正常的每日更新 https://rebase.network/posts/category/web3daily
- 招聘: 20+ 职位 https://github.com/rebase-network/who-is-hiring/issues
- 活动: 支持 9 个活动 https://github.com/rebase-network/event-pool/issues
![2022-10-17 at 15 00](https://user-images.githubusercontent.com/1963646/196110345-70f54905-4252-4144-904b-ee862e868aa3.png)

**Comment by bobjiang** [comment]
Gitcoin China

- 中文周报，http://news.gitcoinchina.com/
- GR16开还是不开？https://gov.gitcoin.co/t/request-for-feedback-proposed-future-for-the-grant-programs-and-gr16-as-we-transition-to-the-protocol/11624

**Comment by Emilyxiannv** [comment]
### DAOSquare 

孵化器产品Phoenix即将上线，目前已经开启第一轮白名单测试：https://twitter.com/DAOSquare/status/1581912702447820802

**Comment by stephanieeee-ho** [comment]
### ECN
DEVCON VI 日程表和深度报道 devcon.app

**Comment by Stamp9** [comment]
### 原语里弄

1. TSS Open Library Kickoff Meeting
2. DeFi 合规组件课题汇报

**Comment by 0x68616f79756e** [comment]
**ETHPlanet**
1.参与支持devcon
2.支持开源社区年会，以及协助分发数字徽章

**Comment by 0x-stan** [comment]
# Dapp-Learning

## Devcon

https://twitter.com/Dapp_Learning/status/1581654718971260928

## repo update

18个PR #838 ～ #856

https://github.com/Dapp-Learning-DAO/Dapp-Learning/pulls

## sharing meeting

近两周举办了2场分享会

- 《dark forest相关》 by ddy
  - https://www.youtube.com/watch?v=z7vcF69y_M8

- 《Frax 项目分享》by Joly


## MIT 公开课翻译(完结)

- MIT class 24 ' Blockchain & Money ' 24. Conclusion
  - https://mp.weixin.qq.com/s/AYTrY9Gi9aVQT4bCbcL4mg
